ipol_demo/modules/core/core/core.py

```diff
@@ -469,117 +72,81 @@ def __init__(self):
-        self.project_folder = cherrypy.config.get("project_folder")
-        self.blobs_folder = cherrypy.config.get("blobs_folder")
+        self.base_url: str = os.environ.get("IPOL_URL", "http://" + socket.getfqdn())
 
-        self.shared_folder_rel = cherrypy.config.get("shared_folder")
+        self.project_folder = os.path.expanduser("~") + "/ipolDevel"
+        self.blobs_folder = (
+            os.path.expanduser("~") + "/ipolDevel/ipol_demo/modules/blobs"
+        )
+
+        self.shared_folder_rel: str = "shared_folder/"
         self.shared_folder_abs = os.path.join(
             self.project_folder, self.shared_folder_rel
         )
-        self.demo_extras_main_dir = os.path.join(
-            self.shared_folder_abs, cherrypy.config.get("demoExtrasDir")
-        )
-        self.dl_extras_dir = os.path.join(
-            self.shared_folder_abs, cherrypy.config.get("dl_extras_dir")
-        )
-        self.share_run_dir_rel = cherrypy.config.get("running.dir")
+        self.demo_extras_main_dir = os.path.join(self.shared_folder_abs, "demoExtras/")
+        self.dl_extras_dir = os.path.join(self.shared_folder_abs, "dl_extras/")
+        self.share_run_dir_rel: str = "run/"
         self.share_run_dir_abs = os.path.join(
             self.shared_folder_abs, self.share_run_dir_rel
         )
 
-        # Security: authorized IPs
-        self.authorized_patterns_file = cherrypy.config.get("authorized_patterns_file")
-        self.authorized_patterns = self.read_authorized_patterns()
-
         # create needed directories
         os.makedirs(self.share_run_dir_abs, exist_ok=True)
         os.makedirs(self.shared_folder_abs, exist_ok=True)
         os.makedirs(self.dl_extras_dir, exist_ok=True)
         os.makedirs(self.demo_extras_main_dir, exist_ok=True)
 
         base_url = os.environ.get("IPOL_URL", "http://" + socket.getfqdn())
-        demorunners_path = cherrypy.config["demorunners_file"]
-        policy = cherrypy.config["dispatcher_policy"]
+        demorunners_path = (
+            os.path.expanduser("~")
+            + "/ipolDevel/ipol_demo/modules/config_common/demorunners.xml"
+        )
+        policy = "lowest_workload"
         self.dispatcher = dispatcher.Dispatcher(
             workload_provider=dispatcher.APIWorkloadProvider(base_url),
             ping_provider=dispatcher.APIPingProvider(base_url),
             demorunners=dispatcher.parse_demorunners(demorunners_path),
             policy=dispatcher.make_policy(policy),
         )
 
-        self.converter = conversion.Converter()
-
-        demoinfo_db = cherrypy.config["demoinfo_db"]
-        demoinfo_dl_extras_dir = cherrypy.config["demoinfo_dl_extras_dir"]
         self.demoinfo = demoinfo.DemoInfo(
-            dl_extras_dir=demoinfo_dl_extras_dir,
-            database_path=demoinfo_db,
+            dl_extras_dir="staticData/demoExtras",
+            database_path="db/demoinfo.db",
             base_url=base_url,
         )
```

Comment:
This is not using the user settings and might lead to inconsistencies with respect to `ipol_demo/modules/core/demoinfo/router.py`.